drafts/draft-brockners-ippm-ioam-geneve.xml

```diff
@@ -424,7 +427,9 @@
           data is present as defined by the Type field, and is defined in
           Section 4 of <xref target="I-D.ietf-ippm-ioam-data"/>.</t>
         </list>Multiple IOAM options MAY be included within the Geneve
-      encapsulation. For example, if a Geneve encapsulation contains two IOAM
+      encapsulation. But more than one IOAM options with the same type MUST NOT
+      be included within the Geneve encapsulation. For example, if a Geneve 
```

Comment:
In case you re-write the paragraph, also use the proper IOAM nomenclature, ie. refer to IOAM-Option-Types instead of the sloppy "IOAM options". 

"Multiple distinct IOAM-Option-Types MAY be included within the same Geneve encapsulation. Each IOAM-Option-Type MUST occur at most once within the same Geneve encapsulation".

Reply:
I have this updated